services/application.go

```diff
@@ -64,22 +66,27 @@ func (app *ChainlinkApplication) Start() error {
 		app.Stop()
 		app.Exiter(0)
 	}()
+	go app.RunManager.Start()
```

Comment:
This could be a normal synchronous call that returns an error but internally feeds runs to the `RunQueue` to be run async. Would be consistent with the other `Start`s